Source/WebKit/UIProcess/API/Cocoa/WKWebViewConfiguration.mm

```diff
@@ -120,11 +120,16 @@ - (void)dealloc
     if (WebCoreObjCScheduleDeallocateOnMainRunLoop(WKWebViewConfiguration.class, self))
         return;
 
-    _pageConfiguration->API::PageConfiguration::~PageConfiguration();
+    [self _protectedPageConfiguration]->API::PageConfiguration::~PageConfiguration();
 
     [super dealloc];
 }
 
+- (Ref<API::PageConfiguration>)_protectedPageConfiguration
+{
+    return *_pageConfiguration;
+}
```

Comment:
Since this is for use inside the class and the source file I might have used a function instead of a method.

Reply:
I wasn't sure how to create a `protectedFoo()` function for an Objective C object. I asked @ddkilzer for help and he recommended adding a method like this. Personally, I'd rather write C++ as much as possible so if there is a way to use a C++ function, that'd be great.

Reply:
I just realized that maybe you can't if the field is private. But I'd just write a free function like:
```
static Ref<API::PageConfiguration > protectedConfiguration(WKWebViewConfiguration *configuration)
{
    return *configuration-> _pageConfiguration;
}
```